.github/workflows/deploy_to_railway.yml

```
@@ -27,11 +27,41 @@ on:
+          
+
+  deploy-to-railway:
+    needs: build-and-push-docker-image
+    runs-on: ubuntu-latest
+    container: ghcr.io/railwayapp/cli:latest
+    environment: 'Development'
+    env:
+      RAILWAY_TOKEN: ${{ secrets.RAILWAY_GITHUB_ACTIONS }}
+    steps:
+      - name: Apply new image to Railway
+        run:  railway up --environment=${{ vars.RAILWAY_ENVIRONMENT_ID }} --service=${{ vars.RAILWAY_SERVICE_ID  }} --detach
```

YAMLlint (1.35.1)

[warning] too many spaces after colon

(colons)